fakestorage/server_test.go

```diff
@@ -13,8 +13,6 @@ import (
 	"strings"
 	"testing"
 
-	"google.golang.org/api/iterator"
-
```

Comment:
oh I meant deleting just the blank line, the import is necessary.

Reply:
ah, alright, I suspected something was going wrong there :D